IFComp/lib/IFComp/Controller/Ballot.pm

```diff
@@ -47,6 +47,21 @@ sub root : Chained('/') : PathPart('ballot') : CaptureArgs(0) {
 sub fetch_entries : Chained('root') : PathPart('') : CaptureArgs(0) {
     my ( $self, $c ) = @_;
 
+    my %rating_for_entry;
```

Comment:
This code block appears twice, identically, in the controller; once in `fetch_entries` and again in `fetch_alphabetized entries`. Could we refactor it into an internal method instead?